engine/graphic/include/Texture.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+#include <GL/glew.h>
+#include <il.h>
+
+class Texture
+{
+public:
+	Texture(char* FileName);
```

Comment:
Let's use string instead of char*
std::string text
If you want to get char* from string, just write text.c_str()